src/main.rs

```diff
@@ -16,7 +16,12 @@ mod utils;
 struct Args {
     /// Bytecode
     #[clap(short, long)]
-    bytecode: String,
+    bytecode: Option<String>,
+
+    /// Bytecode file path
+    #[clap(long, conflicts_with = "bytecode")]
```

Comment:
Could we use an alias here ? short won't work because it will have the same tag as `bytecode`, but maybe we could use `--bf` ?

Reply:
The idea was that user either provides a bytecode or a bytecode file, but makes sense, will refactor this!

Reply:
Done!